sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManager.java

```diff
@@ -37,31 +43,71 @@
      * @param keyStore the keystore.
      */
     public KeyVaultTrustManager(KeyStore keyStore) {
-        this.keyStore = keyStore;
-        if (this.keyStore == null) {
-            try {
-                this.keyStore = KeyStore.getInstance(KeyVaultKeyStore.KEY_STORE_TYPE);
-                this.keyStore.load(null, null);
-            } catch (KeyStoreException | IOException | NoSuchAlgorithmException | CertificateException ex) {
-                ex.printStackTrace();
```

Comment:
Never, never have `printStackTrace()` in our production code.

Reply:
Replaced